.github/workflows/ci.yml

```diff
@@ -63,10 +73,30 @@ jobs:
             gotestfmt --logfile "${AUTHD_TEST_ARTIFACTS_DIR}/gotestfmt.cover.log"
 
           # Convert the raw coverage data into textfmt so we can merge the Rust one into it
-          go tool covdata textfmt -i="${raw_cov_dir}" -o="/tmp/coverage.out"
+          go tool covdata textfmt -i="${raw_cov_dir}" -o="${cov_dir}/coverage.out"
+
+          # Filter out the testutils package
+          grep -v -e "testutils" "${cov_dir}/coverage.out" >"${cov_dir}/coverage.out.filtered"
 
-          # Filter out the testutils package and the pb.go file
-          grep -v -e "testutils" "/tmp/coverage.out" >"/tmp/coverage.out.filtered"
+          # Generate the Cobertura report for Go
+          gocov convert "${cov_dir}/coverage.out.filtered" | gocov-xml > "${cov_dir}/coverage.xml"
+          reportgenerator -reports:"${cov_dir}/coverage.xml" -targetdir:"${codecov_dir}" -reporttypes:Cobertura
```

Comment:
Wondering, can we use `gcovr` instead to convert to cobertura format?

Also I read `reportgenerator` can generate human readable infos, maybe we can print them in a CI report?

Reply:
> Wondering, can we use `gcovr` instead to convert to cobertura format?

`gcovr` is based on C-like coverage structures, so it needs some files that Go does not generate (if I understood correctly).

> Also I read reportgenerator can generate human readable infos, maybe we can print them in a CI report?

This is not necessary, IMO. We already upload the coverage to `codecov` as well (at least we try to) and the tool already adds a report to every pull request and generates a web page with all the necessary information.